Origin: 754658aff38e1bdf487c58bec6174cbecd019d11
Author: dana <dana@dana.is>
Date: Fri Mar 13 09:46:14 2020 -0500
Description: _git: Fix copy/paste error in earlier commit that broke `git stash drop`
 This was broken by workers/44923 (8aa84bd5fc44)
Reviewed-By: Axel Beckert <abe@debian.org>
Comment: This is the upstream commit without the changes to upstream Changelog file.
Bug: https://www.zsh.org/cgi-bin/mla/redirect?WORKERNUMBER=46093

diff --git a/Completion/Unix/Command/_git b/Completion/Unix/Command/_git
index 8487ebc1a..9a44ddbbe 100644
--- a/Completion/Unix/Command/_git
+++ b/Completion/Unix/Command/_git
@@ -1809,7 +1809,7 @@ _git-stash () {
           _nothing
           ;;
         (drop)
-          _arguments -S $endopt \_git-notes
+          _arguments -S $endopt \
             '(-q --quiet)'{-q,--quiet}'[suppress all output]' \
             ':: :__git_stashes' && ret=0
           ;;
